test/tools/router_check/router_check.cc

```diff
@@ -38,6 +38,17 @@ int main(int argc, char* argv[]) {
                   << std::endl;
         return EXIT_FAILURE;
       }
+      const auto cov_file_path = std::getenv("ENVOY_COV_CONFIG_FILE");
```

Comment:
It feels like the `ENVOY_COV_CONFIG_FILE` is not necessary. The tool could have been run in any way and the `failUnder` could have been coming in variety of ways not involving a `.sh` file.

Reply:
Yeah, this is more in an attempt to solve the issue of people not realizing where the value is being set. I'm open to ideas on how to communicate this :)

Reply:
I feel that there are too many knobs, and `ENVOY_COV_CONFIG_FILE` is not tied to anything strongly and could get out of sync easily. I think we should lose this setting. Will that be ok?

Reply:
I'm not particularly tied to it and it is an easy enough thing to add back later. I'll remove it.